include/sequence.h

```diff
@@ -3,132 +3,131 @@
-#define NA_BGM_MIKAU_RIFF 0x58              // Mikau's Tale "MIKAU_LIFF"
-#define NA_BGM_MIKAU_FINALE 0x59            // A single guitar chord. "MIKAU_FIN"
-#define NA_BGM_FROG_SONG 0x5A               // Don Gero's Song
-#define NA_BGM_OCA_SONATA 0x5B              // Ocarina Sonata of Awakening "OCA_AWAKING"
-#define NA_BGM_OCA_LULLABY 0x5C             // Ocarina Goron Lullaby "OCA_LULLABY"
-#define NA_BGM_OCA_NEW_WAVE 0x5D            // Ocarina New Wave Bossa Nova "OCA_TIDE"
-#define NA_BGM_OCA_ELEGY 0x5E               // Ocarina Elegy of Emptiness "OCA_SHELL"
-#define NA_BGM_OCA_OATH 0x5F                // Ocarina Oath to Order "OCA_PLEDGE"
-#define NA_BGM_MAJORAS_LAIR 0x60            // "LAST_DUNGEON"
-#define NA_BGM_OCA_LULLABY_INTRO 0x61       // "OCA_LULLABY_HALF"
-#define NA_BGM_OCA_GUITAR_BASS_SESSION 0x62 // Bass and Guitar Session "OCA_SESSION"
-#define NA_BGM_PIANO_SESSION 0x63           // Piano Solo "DEMO_SESSION"
-#define NA_BGM_INDIGO_GO_SESSION 0x64       // The Indigo-Go's Rehearsal "ALLPART_SESSION"
-#define NA_BGM_SNOWHEAD_TEMPLE 0x65         // Snowhead Temple "SILVER_DUNGEON"
-#define NA_BGM_GREAT_BAY_TEMPLE 0x66        // Great Bay Temple "DEEPBLUE_DUNGEON"
-#define NA_BGM_NEW_WAVE_SAXOPHONE 0x67      // "DEMO_TIDE_SAX"
-#define NA_BGM_NEW_WAVE_VOCAL 0x68          // "DEMO_TIDE_VOCAL"
-#define NA_BGM_MAJORAS_WRATH 0x69           // Majora's Wrath "MUJURA_1"
-#define NA_BGM_MAJORAS_INCARNATION 0x6A     // Majora's Incarnation "MUJURA_2"
-#define NA_BGM_MAJORAS_MASK 0x6B            // Majora's Mask Battle "MUJURA_3"
-#define NA_BGM_BASS_PLAY 0x6C               // Bass Practice "BASS_PLAY"
-#define NA_BGM_DRUMS_PLAY 0x6D              // Drums Practice "DRUMS_PLAY"
-#define NA_BGM_PIANO_PLAY 0x6E              // Piano Practice "PIANO_PLAY"
-#define NA_BGM_IKANA_CASTLE 0x6F            // Ikana Castle
-#define NA_BGM_GATHERING_GIANTS 0x70        // Calling the Four Giants
-#define NA_BGM_KAMARO_DANCE 0x71            // Kamaro's Dance "CAMARO_DANCE"
-#define NA_BGM_CREMIA_CARRIAGE 0x72         // Cremia's Carriage "DONKEY_CART"
-#define NA_BGM_KEATON_QUIZ 0x73             // Keaton "QUIZ"
-#define NA_BGM_END_CREDITS_1 0x74           // The End/Credits I "END_DEMO"
-#define NA_BGM_OPENING_LOOP 0x75            // Forest Ambush (?)		Duplicate.
-#define NA_BGM_TITLE_THEME 0x76             // Title Screen
-#define NA_BGM_DUNGEON_APPEAR 0x77          // Surfacing of Woodfall
-#define NA_BGM_WOODFALL_CLEAR 0x78          // Woodfall Clear "PURIFICATION_0"
-#define NA_BGM_SNOWHEAD_CLEAR 0x79          // Snowhead Clear "PURIFICATION_1"
-#define NA_BGM_INTO_THE_MOON 0x7B           // To the Moon
-#define NA_BGM_GOODBYE_GIANT 0x7C           //
-#define NA_BGM_TATL_AND_TAEL 0x7D           // Tatl and Tael "CHAT"
-#define NA_BGM_MOONS_DESTRUCTION 0x7E       // Moon's Destruction "MOON_VANISH"
-#define NA_BGM_NO_MUSIC 0x7F                //
-#define NA_BGM_END_CREDITS_2 0x82           // The End/Credits II "STAFFROLL2"
```

Comment:
what happened to `NA_BGM_END_CREDITS_2`?

Reply:
It never existed in MM64. It's this song that plays during the remake credits in MM3D: https://www.youtube.com/watch?v=PQxPlwcs0xw

Before removing it, I did try to play it in-game, and unsurprisingly, it just crashed. This *does* make me wonder what the final entry in the table is for `sSeqFlags` in `code_8019AF00.c`. But it certainly isn't this song, since there's no secondary staff roll in the original version of the game.

Reply:
The final entry in `sSeqFlags` could just be padding. Besides, the last entry is `0x7F` i.e. `NA_BGM_NO_MUSIC` so I'm not sure why `NA_BGM_END_CREDITS_2` is commented on it anyway. So that line can go and `sSeqFlags` can just be 126 or 0x7E entries long

Reply:
There was discussion on discord, but just to make sure it's all in this PR:
- Like I said above, there's no second staff roll in MM64, but...
- The credits theme in MM64 *is* split in half, so there are two sequences for it. However...
- It's at 0x7F, *not* 0x82

The code I'm about to push will update this to reflect our new understanding.